### DISCUSSION TITLE
Feature Request: Audit

### DISCUSSION THREAD

**Discussion by NoobieDog** [HotCakeX/Harden-Windows-Security#270] [General]
Hi @HotCakeX 

Thank you for this awesome tool, very nice work. Is there the scope or ability to run the hardening as audit of what is compliant/enabled/disabled before applying fixes for hardening??

Kind regards

**Comment by HotCakeX** [comment]
Hi @NoobieDog 
Thank you very much,
of course, you can use the Harden Windows Security module to audit the system
https://github.com/HotCakeX/Harden-Windows-Security/wiki/Harden%E2%80%90Windows%E2%80%90Security%E2%80%90Module
